index.html

```diff
@@ -0,0 +1,209 @@
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt="" /></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+      <div class="project4">
+        <img src="images/multi.png" alt="" />
+        <div class="proj-desc">
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt="" /></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+    </section>
+    <section class="about-sect">
+      <div class="about">
+        <h2 class="about-intro intro">About <br />Myself</h2>
+        <p class="about-pitch pitch">
+          I'm a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="connect conn-about">
+        <h3>LETS CONNECT</h3>
+        <ul class="social-icons">
+          <li class="icon">
+            <a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+          </li>
+          <!-- <li class="linked"><img src="images/linkedin.png" alt=""></li> -->
+          <li class="icon">
+            <a href=""><i class="fa fa-medium" aria-hidden="true"></i></a>
+          </li>
+          <li class="icon">
+            <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          </li>
+          <li class="icon">
+            <a href=""><i class="fa fa-angellist" aria-hidden="true"></i></a>
+          </li>
+          <li class="icon">
+            <a href=""><i class="fa fa-github" aria-hidden="true"></i></a>
+          </li>
+        </ul>
+      </div>
+      <div>
+        <button class="about-btn see_project" type="button">Get my resume</button>
+      </div>
```

Comment:
- [ ] Kindly the font your are using and the color for social media are not match with the Figma template design.
- [ ] You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design
![Screenshot from 2022-05-28 16-09-01](https://user-images.githubusercontent.com/77842237/170826954-d6af28b1-b56e-4add-a6ab-2760d454e15f.png)

FIgma design
![Screenshot from 2022-05-28 16-09-27](https://user-images.githubusercontent.com/77842237/170826973-d6872d81-730e-46a1-884a-621b55a75d3f.png)